setup.py

```diff
@@ -28,12 +28,20 @@
     namespace_packages=["collective"],
     include_package_data=True,
     zip_safe=False,
-    install_requires=["setuptools", "z3c.jbot",],
-    extras_require={"test": ["plone.app.testing",],},
+    install_requires=[
+        "setuptools",
+        "z3c.jbot",
+    ],
+    extras_require={
+        "test": [
+            "plone.app.testing",
+        ],
+    },
     entry_points="""
       # -*- Entry points: -*-
-
       [z3c.autoinclude.plugin]
       target = plone
+      [console_scripts]
```

Comment:
Please do not add this console script. It will be installed for everyone who adds `collective.honeypot` to their site. If multiple packages do this with the same name, their scripts will override each other. It is only meant for development and you can just use the `update.sh` script for it. `update.py` can be removed as well.
Honestly, I don't think translations are needed. The one translated string should only be seen by spammers, and they are likely just bots, so no human will see it. But yes, I did add this more or less friendly error message because the code may make a mistake and an actual human could see it. So it is okay to have it.

Reply:
i added translations for a different reason: could a bot "learn" that a field is a honeypot because there is the word "spamming" into the alert message?
If yes, with translations we can override that label in our packages without touching the original product.

But if it's overengineered i can remove it completely

Reply:
@mauritsvanrees this is actually the default of add-on's created by plonecli ;)
the update.sh can be deleted, the Python version is more flexible and i use the console script in the self contained buildout all the time. But in a project setup you might be right. Not sure how to solve htis or if it is even a real problem. Because nobody uses update_locales in that context. It only makes sense in a package it self.

Reply:
yes, i copied it from plonecli. Let me know if you want to simplify it here (removing the console script) or not.

Reply:
I would remove it from the console_scripts.

Reply:
ok, removed